docs/source/fitting/fitfunctions/GaussianComptonProfile.rst

```diff
@@ -0,0 +1,42 @@
+.. _func-GaussianComptonProfile:
+
+======================
+GaussianComptonProfile
+======================
+
+.. index:: GaussianComptonProfile
+
+Description
+-----------
+
+The GaussianComptonProfile function describes the Compton profile of a nucleus using a
+Gaussian approximation convoluted with an instrument resolution function,
+that is approximated by a Voigt function. The function approximates the Count rate, :math:`C(t)` as [1],
```

Comment:
This is a small file, so it's not really an issue but perhaps this reference could be interactive:
```suggestion
that is approximated by a Voigt function. The function approximates the Count rate, :math:`C(t)` as in Ref. [#Mayers]_,
```